mlflow/ml-package-versions.yml

```
@@ -255,6 +255,9 @@ fastai:
     minimum: "2.4.1"
     maximum: "2.7.17"
     requirements:
+      # TODO: fastai depends on spacy, but spacy>=3.8.2 requires python>=3.9
+      #  Once migrating fastai tests to python 3.9, we should remove spacy<3.8.2
```

```suggestion
      # Once migrating fastai tests to python 3.9, we should remove spacy<3.8.2
```
Let's also link the filed issue here :)